backend/src/controller/account_manage.rs

```diff
@@ -57,35 +62,69 @@ pub async fn account_registration(
         (status = 500, description = "エラー発生時", body = CommonErrorResponseBody),
     )
 )]
-// ログインapi
+/**
+ * ログインapi
+ * ログインIDとパスワードを受け取り、認証を行う
+ * 認証に成功した場合は、セッションIDをCookieにセットして返す
+ * TODO: バリデーション失敗時、とログイン失敗時のレスポンスタイムを同一にするためにコントロールする
+ * TODO: 生成したtokenをDBに保存する
+ */
 #[post("/auth/login")]
 pub async fn auth_login(
     data: web::Data<AppState>,
     request: web::Json<LoginRequest>,
 ) -> Result<HttpResponse, Error> {
+    let validation_result: &str = validation_account_manage(&request);
+
+    if validation_result != "" {
+        return core::result::Result::Ok(HttpResponse::BadRequest().json(LoginResponse {
+            message: validation_result.to_string(),
```

Comment:
`.to_owned()`か`.to_string()`に統一しましょう
(確か`.to_owned()`に統一することに決まった記憶があります)
```suggestion
            message: validation_result.to_owned(),
```